### ISSUE TITLE
Package Control: The dependency 'jsonschema' is not available

### ISSUE THREAD

**Issue by shtetik** [wbond/package_control#1402]
Hello!
Error installing dependencies when trying to install SublimeLinter.
The rest of the package is installed without problems.
```
Package Control: The dependency 'jsonschema' is not currently installed; installing...
Package Control: The dependency 'jsonschema' is not available
Package Control: The dependency 'jsonschema' could not be installed or updated
```

**Comment by FichteFoll** [comment]
Does it work when you try again? I don't see anything out of the ordinary; that dependency exists and does not have a platform restriction (only a restriction for ST3 but SL only requests that for ST3 anyway).

**Comment by rehat101** [comment]
I also get the same error. 
Tried multiple times.

**Comment by gaoyicoder** [comment]
I also get the same error.

**Comment by gaoyicoder** [comment]
> Hello!
> Error installing dependencies when trying to install SublimeLinter.
> The rest of the package is installed without problems.
> 
> ```
> Package Control: The dependency 'jsonschema' is not currently installed; installing...
> Package Control: The dependency 'jsonschema' is not available
> Package Control: The dependency 'jsonschema' could not be installed or updated
> ```

I just solved the problem by updating curl from 7.52.1 to 7.63.0

**Comment by shtetik** [comment]
> > Hello!
> > Error installing dependencies when trying to install SublimeLinter.
> > The rest of the package is installed without problems.
> > ```
> > Package Control: The dependency 'jsonschema' is not currently installed; installing...
> > Package Control: The dependency 'jsonschema' is not available
> > Package Control: The dependency 'jsonschema' could not be installed or updated
> > ```
> 
> I just solved the problem by updating curl from 7.52.1 to 7.63.0

Thanks, it helped me too.
